src/Transport/Http.php

```diff
@@ -87,7 +94,16 @@ public function exec(Request $request, array $params): Response
         /* @see Connection::setConnectTimeout() */
         $connectTimeout = $connection->getConnectTimeout();
         if ($connectTimeout > 0) {
-            \curl_setopt($conn, \CURLOPT_CONNECTTIMEOUT, $connectTimeout);
+            // If cURL >= 7.16.2, timeout can be expressed in ms
+            if (\defined('CURLOPT_CONNECTTIMEOUT_MS')) {
```

Comment:
We can probably remove this and just assume that curl is greater than `7.16.2` which was released a long way back (April 11 2007).